beetsplug/smartplaylist.py

```diff
@@ -55,13 +57,13 @@ def __init__(self):
         )
 
         self.config["prefix"].redact = True  # May contain username/password.
-        self._matched_playlists = None
-        self._unmatched_playlists = None
+        self._matched_playlists: set[tuple[Any, Any, Any]] = set()
```

Comment:
Any way to actually type the playlists? The Any usage here makes me a bit uncomfortable.